bamboo/docker-compose.yml

```diff
@@ -41,10 +41,11 @@ services:
     environment:
       ES_JAVA_OPTS: "-Xms750m -Xmx750m"
   localstack:
-    image: maven.earthdata.nasa.gov/localstack/localstack:0.10.7
+    image: maven.earthdata.nasa.gov/localstack/localstack:0.11.5
     network_mode: "service:build_env"
     environment:
-      SERVICES: "cloudformation,cloudwatch,cloudwatchlogs,dynamodb,kinesis,kms,lambda,s3,secretsmanager,sns,sqs,stepfunctions,ssm"
+      SERVICES: "cloudformation,cloudwatch,cloudwatchlogs,dynamodb,iam,kinesis,kms,lambda,s3,secretsmanager,sns,sqs,stepfunctions,ssm,logs"
+      EAGER_SERVICE_LOADING: "1"
```

Comment:
Is `EAGER_SERVICE_LOADING` available on version 0.11.5?

Reply:
Indeed not. Good catch

Reply:
6e47aae4a2738242ee5e49192a3b3467e0d73808